README.md

```diff
@@ -16,97 +22,70 @@
 
   [Giptops - maintained by gitops operator](./openshift-gitops/apps/charts/values.yaml)
 
-# Implementation
+## Implementation
 
-## Set up the infra 
-### Install the operator
+### Set up the infra
+
+#### Install the operator
 
 ```bash
-oc apply -f openshift-gitops/infra/gitops-operators.yaml
-```
-### Monitor the operator install
-```bash
-oc get csv -n openshift-gitops -w
-```
-### Delete the default app installed by the operator
-```bash
-oc delete argocd openshift-gitops -n openshift-gitops
-```
-### Deploy a argocd app with vault plugin
-```bash
-oc apply -f openshift-gitops/infra/argocd.yaml
+$ ./scripts/infra/argocd.sh provision
+subscription.operators.coreos.com/openshift-gitops-operator created
+sleep 10 seconds until argocd subscription is installed.
+Operator subscription installed successfully
+Deploy ArgoCD app
+serviceaccount/argocd-repo-server created
+argocd.argoproj.io/redhat-kong-gitops created
+sleep 10 seconds until argocd app is Available.
+ArgoCD app deployed successfully
 ```
 
-### Get the route and password of argocd gui
+#### Get the route and password of argocd gui
+
 ```bash
 oc get routes -n openshift-gitops redhat-kong-gitops-server --template='{{ .spec.host }}'
 ```
+
 ```bash
 oc get secret -n openshift-gitops redhat-kong-gitops-cluster -ojsonpath='{.data.admin\.password}' | base64 -d
 ```
 
-### Add controlplane and dataplane cluster
+#### Add controlplane and dataplane clusters
+
 ```bash
-export ARGOCD_SERVER_URL=$(oc get routes -n openshift-gitops | grep redhat-kong-gitops-server | awk '{print $2}')
 oc get secret -n openshift-gitops redhat-kong-gitops-cluster -ojsonpath='{.data.admin\.password}' | base64 -d
-argocd login $ARGOCD_SERVER_URL
-argocd cluster add dp
-argocd cluster add cp
+argocd login `oc get routes -n openshift-gitops redhat-kong-gitops-server --template='{{.spec.host}}'`
+argocd cluster add -y --name dp <dp-context>
```

Comment:
This command is giving the following error even though I login to argo server successfully 
`FATA[0000] Failed to create service account "argocd-manager" in namespace "kube-system": Unauthorized `

Reply:
please, replace the placeholder `<dp-context>` by your context, e.g. `kind-9545298d-0e39-4aee-aafb-4305ad05a4fe`. To list all the configured contexts you can just use `argocd cluster add`